stingray/modeling/parameterestimation.py

```diff
@@ -297,34 +290,73 @@ def fit(self, lpost, t0, neg=True, scipy_optimize_options=None):
             params = [getattr(newmod,name) for name in newmod.param_names]
             bounds = [p.bounds for p in params if not np.any([p.tied, p.fixed])]
 
+            if len(bounds) > 0 and self.fitmethod not in ["L-BFGS-B",
+                                                          "TNC",
+                                                          "SLSQP"]:
+                logging.warning("Fitting method %s "%self.fitmethod +
+                                "cannot incorporate the bounds you set!")
+
+            if len(bounds) == 0 or self.fitmethod not in ["L-BFGS-B",
```

Comment:
well, the problem is that it spits out a warning *every* time you try to fit something. For some of the Monte Carlo methods, where we simulate and fit power spectra, those warnings get really annoying really fast. 

That's pretty much the main reason. If you have a better solution, I'm happy to use that.